### DISCUSSION TITLE
Error While starting DB with Gremlin Plugin

### DISCUSSION THREAD

**Discussion by Praveenstein** [ArcadeData/arcadedb#653] [Q&A]
Hi all, thanks for the great work at arcadedb, I'm a java rookie here, and i'm trying to start the db server with the gremlin server plugin, but i'm getting "Error: Could not find or load main class GremlinServer:com.arcadedb.server.gremlin.GremlinServerPlugin", "Caused by: java.lang.ClassNotFoundException: GremlinServer:com.arcadedb.server.gremlin.GremlinServerPlugin", please do help me, thanks
![Screenshot 2022-12-01 113736](https://user-images.githubusercontent.com/45847217/204978790-abc127a0-2a60-4f0f-9c63-1e1de8fedb2f.png)

**Comment by lvca** [comment]
It looks like the `server.bat` is not working properly. Instead of displaying the ArcadeDB welcome message

```
echo ""
echo " █████╗ ██████╗  ██████╗ █████╗ ██████╗ ███████╗██████╗ ██████╗ "
echo "██╔══██╗██╔══██╗██╔════╝██╔══██╗██╔══██╗██╔════╝██╔══██╗██╔══██╗"
echo "███████║██████╔╝██║     ███████║██║  ██║█████╗  ██║  ██║██████╔╝"
echo "██╔══██║██╔══██╗██║     ██╔══██║██║  ██║██╔══╝  ██║  ██║██╔══██╗"
echo "██║  ██║██║  ██║╚██████╗██║  ██║██████╔╝███████╗██████╔╝██████╔╝"
echo "╚═╝  ╚═╝╚═╝  ╚═╝ ╚═════╝╚═╝  ╚═╝╚═════╝ ╚══════╝╚═════╝ ╚═════╝ "
```

It displays weird characters. 

Then it doesn't use the arguments you specified when it's calling the server:

```
:setArgs
if ""%1""=="""" goto doneSetArgs
set CMD_LINE_ARGS=%CMD_LINE_ARGS% %1
shift
goto setArgs

:doneSetArgs
```

Anybody has Windows can help on this?

In the meantime, could you please try setting this:

```
set ARCADEDB_SETTINGS=-Darcadedb.server.plugins=GremlinServer:com.arcadedb.server.gremlin.GremlinServerPlugin
```

And then just start the server with `server.bat`

**Reply by Praveenstein** [reply]
Thanks for the quick response, so tried the set method and started the server, but getting a similar error (I've attached the screenshots), but if unset the variable and do not start the gremlin server, everything works fine (attached the screenshots below)
![Screenshot 2022-12-02 101603](https://user-images.githubusercontent.com/45847217/205217597-28b6bc1f-ccef-4ff3-b5c8-db3364e129db.png)
![Screenshot 2022-12-02 101618](https://user-images.githubusercontent.com/45847217/205217601-c44fa73c-4958-4cf1-a349-133c3aba4f30.png)
![Screenshot 2022-12-02 101649](https://user-images.githubusercontent.com/45847217/205217602-8c4e5549-dd17-45f0-80b2-e88684b6faaa.png)
![Screenshot 2022-12-02 102055](https://user-images.githubusercontent.com/45847217/205217603-a4661d92-5c7b-4988-aa18-8561ce3e8e0e.png)
![Screenshot 2022-12-02 102113](https://user-images.githubusercontent.com/45847217/205217604-6ac92964-9474-4896-9037-dfcb06acdac3.png)

**Reply by lvca** [reply]
Try to start the server from the main directory with `bin\server.bat`

**Reply by Praveenstein** [reply]
Thanks, worked like a charm (but still displays a weird character instead of arcade DB)

**Reply by lvca** [reply]
I've removed the banner in `server.bat`, hoping somebody will come with a working one.
